nautobot_ssot_infoblox/diffsync/models/nautobot.py

```diff
@@ -138,21 +290,70 @@ def get_vlan_status(status: str) -> str:
 
     def update(self, attrs):
         """Update VLAN object in Nautobot."""
-        _vlan = OrmVlan.objects.get(vid=self.vid)
+        _vlan = OrmVlan.objects.get(id=self.pk)
         if attrs.get("status"):
             _vlan.status = OrmStatus.objects.get(name=self.get_vlan_status(attrs["status"]))
         if attrs.get("description"):
             _vlan.description = attrs["description"]
-        _vlan.validated_save()
+        if "ext_attrs" in attrs:
+            self.process_ext_attrs(diffsync=self.diffsync, vlan=_vlan, extattrs=attrs["ext_attrs"])
+        if not _vlan.group.site and _vlan.site:
+            _vlan.group.site = _vlan.site
+            _vlan.group.validated_save()
```

Comment:
Doesn't appear you need to do this here as it's handled with error handling right below.

Reply:
The ValidationError bit I added specifically when trying to fix the issue I ran into. Basically, I ran into an error being thrown when a VLAN didn't have a Site that matched the VLAN Group. Adding this bit helped ensure it was set on both so the ValidationError wouldn't be thrown. If there's a better way of doing this I'm all ears.